scripts/index.js

```diff
@@ -0,0 +1,90 @@
+// Description:
+//   Report errors/alerts from third party services
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   HUBOT_IRC_SERVER
+//   HUBOT_IRC_ROOMS
+//   HUBOT_IRC_NICK
+//   HUBOT_IRC_PORT - Optional
+//   HUBOT_IRC_USERNAME - Optional
+//   HUBOT_IRC_PASSWORD - Optional
+//   HUBOT_IRC_NICKSERV_PASSWORD - Optional
+//   HUBOT_IRC_NICKSERV_USERNAME - Optional
+//   HUBOT_IRC_SERVER_FAKE_SSL - Optional
+//   HUBOT_IRC_SERVER_CERT_EXPIRED - Optional
+//   HUBOT_IRC_UNFLOOD - Optional
+//   HUBOT_IRC_DEBUG - Optional
+//   HUBOT_IRC_USESSL - Optional
+//   HUBOT_IRC_PRIVATE - Optional
+//   HUBOT_IRC_USESASL - Optional
+//
+// Commands:
+//   None
+//
+// Notes:
+//   None
+//
+// Author:
+//   helfi92
+module.exports = function(robot) {
+  const alertFromRequest = (req) => {
+    let alert = req.body.payload ? req.body.payload : req.body;
+
+    if (typeof alert === 'string') {
+      alert = JSON.parse(alert);
+    }
+
+    return alert;
+  };
+
+  const messageRooms = (message) => {
+    const ircRooms = process.env['HUBOT_IRC_ROOMS'];
+
+    if (!ircRooms) {
+      return;
+    }
+
+    const rooms = ircRooms.split(',[\s]*');
+
+    rooms.forEach(room => robot.messageRoom(room, message));
+  };
+
+  robot.router.post('/hubot/sentry', function(req, res) {
+    const data = alertFromRequest(req);
+    const message = `[Sentry] Alert from ${data.project} ${data.project_name}\n` +
```

Comment:
Consider using a single template string and something like the common-tags package's `stripIndents` in the future.

Reply:
@eliperelman any idea how to create multiline IRC messages. This currently sends a new message for each line. Not the best UX.

Reply:
I'm not sure. Maybe there is a way to put all the relevant data in a single line, with a link to details?